### DISCUSSION TITLE
Welcome to MODFLOW 6 Discussions!

### DISCUSSION THREAD

**Discussion by christianlangevin** [MODFLOW-ORG/modflow6#1160] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our MODFLOW community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by JJHelly** [comment]
Aloha.  I'm a newb trying to run the  jupyter notebook examples but cannot get past this error.  I'm not a python expert but would say somewhat more than a novice.  Any suggestions how to get past this problem:

AttributeError: module 'config' has no attribute 'base_ws'

Apparently I don't know how to access the config.py code in the common directory.
Mahalo.
[python-problem.pdf](https://github.com/MODFLOW-USGS/modflow6/files/11583275/python-problem.pdf)

**Comment by JJHelly** [comment]
Aloha.  Nevermind.  Solved by uninstalling a conflicting site package.

**Comment by NTimGlover** [comment]
Thanks for all your hard work. (You and USGS in general).

A suggestion.  I need to validate an installation of MODFLOW for a project I'm doing.  MODFLOW-2005's examples included the output (*.lst files) from the runs so I could use them as known-good files to check against when I installed software and ran the examples. However, MODFLOW 6.5.0 win64 install no longer includes the results files.  Would you please consider adding the results files to the examples in the distribution?  Or make them available elsewhere (maybe here?)
